src/main.rs

```diff
@@ -373,6 +455,11 @@ Options:
     -b, --for-bower
         Use bower.json instead of package.json
 
+   -N, --allow-npm-dev-deps <module1,module2,...> When using --for-bower, this
```

Comment:
I couldn't see the code that takes the argument `<module1,module2,...>`, is this documentation for a previous version of the api?

Reply:
woops, yes it is ^_^